docs/source/analyze/weave-report.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Weave Report
+description: Analyze weave reports
+---
+
+The weave analyzer runs several checks to detect problems with the [weave](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/) CNI provider.
+
+## Paramaters
+
+**reportFileGlob:** Filepath in the support bundle for collected weave reports.
+
+## Example Analyzer Definition
+
+```yaml
+apiVersion: troubleshoot.sh/v1beta2
+kind: SupportBundle
+metadata:
+  name: weave-sample
+spec:
+  collectors:
+    - exec:
+        collectorName: weave-report
+        command:
+          - /home/weave/weave
+        args:
+          - --local
+          - report
+        containerName: weave
+        exclude: ""
+        name: kots/kurl/weave
+        namespace: kube-system
+        selector:
+          - name=weave-net
+        timeout: 10s
+
+  analyzers:
+    - weaveReport:
+        reportFileGlob: 'kots/kurl/weave/kube-system/*/weave-report-stdout.txt'
+```
+
+## Included Analyzers
+
+### IPAM Pool Utilization
+
+A warning will be generated when at least 85% of the available IPs in the weave subnet are in use by pods.
+
+### IPAM Pending Allocation
+
+A warning will be generated when there are pods waiting to be allocated an IP.
```

Comment:
Can we expand here at all? I could see people asking if this is immediate  or if it will appear after a certain pod status or amount of time or something? If not then this is OK as is.

Reply:
Added description